CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
 
+* `CHORE`: don't hide overlays on canvas move by default ([#798](https://github.com/bpmn-io/diagram-js/issues/798))
```

Comment:
Let's make this a feature?

```suggestion
* `FEAT`: don't hide overlays on canvas move by default ([#798](https://github.com/bpmn-io/diagram-js/issues/798))
```

Reply:
agreed, done in [380348c](https://github.com/bpmn-io/diagram-js/pull/799/commits/380348c9613602310a4d5b6c400166bd3a9c4a5a)